doc/Requirements Specification/Parts/Purpose.lyx

```diff
@@ -849,7 +849,7 @@ gls{Beagle}
 
 \end_inset
 
- provides the user to control the analysis with a 
+ provides users to control the analysis with a 
```

Comment:
Am I getting tired or does this sentence make no sense at all?

How about:

“Beagle offers users a GUI to control the analysis”